array-foreach/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for array-foreach
+// Project: https://www.npmjs.com/package/array-foreach
+// Definitions by: Steve Jenkins <https://github.com/skysteve>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+declare module 'array-foreach' {
```

Comment:
could you strip `declare module 'array-foreach' {`?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred